### PR TITLE
Add label fqn description

### PR DESCRIPTION
**Pull request by vterentev**
## Changes

Add label FQN page. 
Add reference to label fqn and label fqn examples to all label pages

## Checklist

- [-] My article available in Navigation sidebar and in root article of selected category ([how to do it in README](https://github.com/santiment/academy#add-an-article-into-navigation-sidebar))
- [x]	My article have [metadata](https://github.com/santiment/academy#metadata) (title, description, date when updated and author)
- [x]	All added/updated links in article are exist, images shows correctly

### CODE REVIEW

File: src/docs/labels/label-fqn/index.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+author: Vladimir
+date: 2024-04-09
+description: Labels usage in API
+---
+
+## Description
+This page outlines the format and use of string identifiers for labels, which are primarily utilized in our API to reference labels. These identifiers can also be found in Sanbase.
+
+### General Structure
+A typical label identifier follows the format `santiment/owner->binance:v1`:
```
Comment (IvanIvanoff) [inline src/docs/labels/label-fqn/index.md:12]:
```suggestion
A typical label identifier follows the format `label_owner/key->value:version`. For example: `santiment/owner->binance:v1`.
```

File: src/docs/labels/label-fqn/index.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
+---
+title: Label-fqn
```
Comment (IvanIvanoff) [inline src/docs/labels/label-fqn/index.md:2]: The title should be human readable, i.e. `Label FQN`.
Also, let's also put in brackets what `fqn` means.

File: src/docs/labels/label-fqn/index.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+author: Vladimir
+date: 2024-04-09
+description: Labels usage in API
```
Comment (IvanIvanoff) [inline src/docs/labels/label-fqn/index.md:5]: Not necessarily in the API -- it applies to Sanbase, Queries and API.

File: src/docs/labels/label-fqn/index.md [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+- `owner`: Denotes the key of the label. Our academy provides a comprehensive list of all available labels.
+- `binance`: Represents the value of the label. Suitable for labels that possess values, this information can be specified within each label in the academy.
+- `v1`: We employ versioning when assigning labels with specific algorithms to ensure consistency. When an algorithm is updated, we introduce a new version to accommodate changes, while still providing access to labels generated by previous versions.
+
+### Labels Associated with Assets
+For labels like whale or whale_usd_balance, there's the option to include an asset in the identifier. For example, to identify addresses labeled as whale within the EtherFi asset ether-fi-ethfi, the label identifier would be santiment/whale(ether-fi-ethfi):v1.
+
+### Labels Featuring Key->Value Pairs
+To obtain addresses associated with specific values, such as owner, you can also specify the value, like Binance. For instance, a label identifier for addresses belonging to Binance would be `santiment/owner->binance:v1`.
```
Comment (IvanIvanoff) [inline src/docs/labels/label-fqn/index.md:22]:
```suggestion
The `value` part of the key can be used to obtain the addresses that are associated with that value.
For example `santiment/centralized_exchange:v1` will select any address that is owned by a CEX.
To obtain the addresses of Binance, use the `owner` key and `binance` value: `santiment/owner->binance:v1`.
```

**Review by alresharin — APPROVED** [review]

**Pull request merged**